src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/TableFormat/DetailedTableFormatStrategy.cs

```diff
@@ -8,9 +8,9 @@ internal class DetailedTableFormatStrategy : ITableFormatStrategy
         private readonly string[] _detailedVerbosityTableHeader = { "Package ID", "Latest Version", "Owners", "Total Downloads", "Vulnerable", "Deprecation", "Project URL", "Description" };
         private readonly int[] _detailedColumnsToHighlight = { 0, 2, 6, 7 };
 
-        public Table CreateTable()
```

Comment:
Similar feedback as in the WrappingTable class. 
If you keep the type Table, then none of these changes are needed.